Interop/Indexer/src/main/kotlin/org/jetbrains/kotlin/native/interop/indexer/J2ObjCParser.kt

```diff
@@ -104,6 +103,26 @@ class J2ObjCParser: ClassVisitor(Opcodes.ASM5) {
     }
   }
 
+  /**
+   * Parses the methods in the class to find getters/setters and generates properties accordingly
+   *
+   * @param methods List of ObjCMethods in the class
+   * @return List of ObjCProperty based off of the getters/setters in class
+   */
+  private fun parseForProperties(methods: List<ObjCMethod>): List<ObjCProperty> {
+    val getters = methods.filter{(it.selector.startsWith("get")) && it.parameters.isEmpty()}
+
+    return getters.map{
+      val varName = it.selector.substring(3)
+      val setter = methods.find{it.selector.startsWith("set${varName}") && it.parameters.size == 1}
```

Comment:
Style question: Does it make sense to define these variables or should I just move it inline into the ObjCProperty definition?

Reply:
I think this makes sense here but it's really subjective and there's no hard-and-fast rule.  I tend to inline if it still fits a single line, personally.

While we're talking about style, note that conventionally there's spaces around { and }, so "methods.find { it... }"